src/Makefile.am

```diff
@@ -260,6 +267,13 @@ gs-shell-search-provider-generated.h gs-shell-search-provider-generated.c: Makef
 		--generate-c-code gs-shell-search-provider-generated \
 		$(srcdir)/shell-search-provider-dbus-interfaces.xml
 
+gs-discovery-feed-content-provider-generated.h gs-discovery-feed-content-provider-generated.c: Makefile.am $(srcdir)/discovery-feed-content-provider-dbus-interfaces.xml
+	$(AM_V_GEN) gdbus-codegen \
+	 	--interface-prefix com.endlessm. \
```

Comment:
Hmm, I don't see anything off here, at least not compared to the shell-search-provider line above